app/assets/stylesheets/config/_generator.scss

```diff
@@ -34,7 +34,8 @@
   (
     'opacity',
     'opacity',
-    ('100': 1, '75': 0.75, '50': 0.5, '25': 0.25, '0': 0),
+    // opacity 0 is 0.00001 because of ChromeVox. See https://github.com/forem/forem/issues/12939
```

Comment:
```suggestion
    // Explicitly set opacity-0 to 0.00001 to ensure accessibility on ChromeVox. See https://github.com/forem/forem/issues/12939
```

Reply:
Thanks for the impovement to the comment @vaidehijoshi! ❤️